web_app/tests/test_dashboard.py

```diff
@@ -265,28 +307,32 @@ async def test_external_service_errors(mock_db_connector):
         MOCK_CONTRACT_ADDRESS
     )
     mock_db_connector.get_positions_by_wallet_id.return_value = [MOCK_POSITION]
-    DashboardMixin.get_wallet_balances = AsyncMock(
-        side_effect=Exception("External API error")
-    )
-    with pytest.raises(Exception) as exc_info:
-        await get_dashboard(VALID_WALLET_ID)
-    assert str(exc_info.value) == "External API error"
-
-
-@pytest.mark.asyncio
-async def test_zklend_service_error(mock_db_connector):
-    """Test handling of ZkLend service failure."""
-
-    mock_db_connector.get_contract_address_by_wallet_id.return_value = (
-        MOCK_CONTRACT_ADDRESS
-    )
-    mock_db_connector.get_positions_by_wallet_id.return_value = [MOCK_POSITION]
-
-    DashboardMixin.get_wallet_balances = AsyncMock(return_value=MOCK_WALLET_BALANCES)
-    DashboardMixin.get_zklend_position = AsyncMock(
-        side_effect=Exception("ZkLend API error")
-    )
-    with pytest.raises(Exception) as exc_info:
-        await get_dashboard(VALID_WALLET_ID)
-
-    assert str(exc_info.value) == "ZkLend API error"
+    with patch(
+        "web_app.contract_tools.mixins.health_ratio.HealthRatioMixin.get_health_ratio_and_tvl",
+        side_effect=Exception("External API error"),
+    ) as mock_get_health_ratio_and_tvl:
+        with pytest.raises(Exception) as exc_info:
+            await get_dashboard(VALID_WALLET_ID)
+        assert str(exc_info.value) == "External API error"
+
+
+# @pytest.mark.asyncio
+# async def test_zklend_service_error(mock_db_connector):
```

Comment:
It was commented, because it was failed
could you please uncomment and check it it's passed

Reply:
Because I dont see the zklenr service called in the ‘get_dashboard’ imp @djeck1432

Reply:
then I will remove it, I recalled that we reworked that logic a bit